### ISSUE TITLE
Potential improvements for User Guide

### ISSUE THREAD

**Issue by macareonie** [macareonie/ped#11]
### The user guide lacks certain details at parts and could have better ordering of features.

Imagining an example workflow could help with ordering of features. 
Example: you open the app -> you add multiple persons -> you find a specific person -> you list all persons  

It might make more sense to have the features in a similar order: add, addAttributes, find, list...
instead of having list at the start. Could just be a nit though.

### Another problem would be the lack of details for certain features. 

The add command format shows add /[Attr name] [Attr value] but it isn't clear what exactly are the possible attributes that can be added. Only further down the UG does it specify that some attributes already have fixed value types and Attributes can also be user-defined etc. Would be nice if there was a some information at the start that explains all of this beforehand.

As stated above, details for relations could also use some fine-tuning. This includes: stating that relations too can be user-defined, prefacing that there are two types of relations, role-less and role-based, possibly separating them into 2 different features etc.

### TLDR
- Ordering of features could be changed (if applicable)
- Certain features require more details
